index.html

```diff
@@ -173,7 +173,7 @@ <h2>
       </dl>
       <p>
         The <a>visibility states</a> are reflected in the API via the
-        <a>VisibilityState</a> enum.
+        {{VisibilityState}} enum.
```

Comment:
Do you have a guide on how one differs from the other?

Reply:
you mean `[=Thing=]` and `{{Thing}}`? Full guide coming soon... but it's  [=Thing=] is a concept and {{Thing}} is IDL (same micro-syntax that BikeShed uses).

Reply:
and what's `<a>Thing</a>` compared to that?

Reply:
so, the concept of a "URL" vs the {{URL}} IDL interface, for instance.

Reply:
You can search for terms via this https://respec.org/xref/ :)

Reply:
That shows you if something is concept, an event, an exception, etc. and how to cite it... and which spec it comes from.

Reply:
Cool, thanks! :)

Reply:
Note, you can now access the xref UI directly from within the spec via the ReSpec pill:

![Screenshot_2019-08-01_21_29_50](https://user-images.githubusercontent.com/870154/62290009-97258c00-b4a3-11e9-937c-626e7ea77197.png)

Reply:
Sorry, didn't answer your question exactly: 

> and what's <a>Thing</a> compared to that?

that's just an alternative (old) syntax... backwards compatible, but less clear.